packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -89,20 +89,20 @@ export type ShimmerElementTypes = ShimmerCircleElement | ShimmerRectElement;
 export interface ShimmerTokens extends IBackgroundColorTokens {
   /**
    * Specifies the Shimmer effect angle in degrees (produced by a gradient)
-   * @defaultValue 0
+   * @defaultValue 45 on Android, 0 on other platforms
    */
   angle?: number;
 
   /**
    * Specifies the animation delay time in milliseconds
-   * @defaultValue 0
+   * @defaultValue 500 on win32, 0 on other platforms
    */
   delay?: number;
 
   /**
    * Specifies the time required to traverse the control in milliseconds
    *
-   * @defaultValue 2000
+   * @defaultValue 1000 on Android, 2000 on win32, 7000 on other platforms
```

Comment:
Why are the defaults different across platforms? I thought these all basically had the same designs?

Reply:
For Android this was based on design guidelines.
While existing code for win32 and other platforms had the value 2000 and 7000 respectively.